### PR TITLE
Refactoring percent-fee mode tests

### PR DESCRIPTION
**Pull request by ngadiyak**

### CODE REVIEW

File: dex-it/src/test/scala/com/wavesplatform/it/sync/OrderFeeTestSuite.scala [hunk 1]
```diff
@@ -934,6 +939,20 @@ class OrderFeeTestSuite extends MatcherSuiteBase {
         node.cancelOrder(alice, ethWavesPair, orderId)
         node.reservedBalance(alice) shouldBe empty
       }
+
+      docker.restartNode(node, ConfigFactory.parseString("waves.dex.order-fee.mode = percent\nwaves.dex.order-fee.percent.min-fee = 0.1\nwaves.dex.order-fee.percent.asset-type = price"))
+
+      withClue("in percent-fee mode fee asset id should not always be Waves for V1 order") {
```
Comment (msmolyakov) [inline dex-it/src/test/scala/com/wavesplatform/it/sync/OrderFeeTestSuite.scala]: **should** always be?

File: dex-it/src/test/scala/com/wavesplatform/it/sync/OrderFeeTestSuite.scala [hunk 2]
```diff
@@ -934,6 +939,20 @@ class OrderFeeTestSuite extends MatcherSuiteBase {
...
+        assertBadRequestAndMessage(
+          node.placeOrder(sender = alice, pair = wavesUsdPair, orderType = OrderType.BUY, amount = 100, price = 100000000L, fee = 10, version = 1: Byte, feeAsset = IssuedAsset(UsdId)),
+          unexpectedFeeAsset(UsdId))
+      }
+
+      withClue("in percent-fee mode fee asset id should not always be Waves for V2 order") {
```
Comment (msmolyakov) [inline dex-it/src/test/scala/com/wavesplatform/it/sync/OrderFeeTestSuite.scala]: **should** always be?

Comment (ngadiyak) [reply dex-it/src/test/scala/com/wavesplatform/it/sync/OrderFeeTestSuite.scala]: +

File: dex-it/src/test/scala/com/wavesplatform/it/sync/OrderFeeTestSuite.scala (hunk 1 above)
Comment (ngadiyak) [reply dex-it/src/test/scala/com/wavesplatform/it/sync/OrderFeeTestSuite.scala]: fixed

File: dex-it/src/test/scala/com/wavesplatform/it/sync/OrderFeeTestSuite.scala (hunk 2 above)
Comment (ngadiyak) [reply dex-it/src/test/scala/com/wavesplatform/it/sync/OrderFeeTestSuite.scala]: fixed

**Pull request merged**